### PR TITLE
Intro improvements, volumeSize references, default volume size 80G.

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Description
Was going through old issues, saw https://github.com/weaveworks/eksctl/issues/780 and also noticed the intro docs were a little unorganized. WDYT?

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [x] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: userdocs/src/introduction.md [hunk 1]
```diff
@@ -113,26 +139,7 @@ eksctl create cluster --node-volume-size=50 --node-volume-type=io1
 
 ```
 
-!!! note
```
Comment (martina-if) [inline userdocs/src/introduction.md:116]: This is still true for us-east-1 and many users still experience issues there.

Comment (michaelbeaumont) [reply userdocs/src/introduction.md:116]: OK, do you think it should be in the introduction?

Comment (martina-if) [reply userdocs/src/introduction.md:116]: Perhaps here https://eksctl.io/usage/creating-and-managing-clusters/ ?

Comment (michaelbeaumont) [reply userdocs/src/introduction.md:116]: @martina-if done, WDYT?

**Comment by michaelbeaumont** [comment]
@martina-if Also added note about default volume size (missing from the last PR)

**Pull request merged**